source/renderer/app/api/api.js

```diff
@@ -375,10 +374,10 @@ export default class AdaApi {
         spendingPassword,
       };
       const response: Transaction = await createTransaction(this.config, { data });
-      Logger.debug(`AdaApi::createTransaction success: ${stringifyData(response)}`);
+      Logger.debug('AdaApi::createTransaction success', { ...response });
```

Comment:
@MarcusHurney why do we use `{ ...response }` in some places and `{ response }` in others?

Reply:
@nikolaglumac 
I was originally using `{ ...response }` to prevent the data object in the logs from becoming nested too many layers deep, but then decided it was more explicit to name the response data when possible, for example like this: `{ wallets: response }`. I've gone through and changed the instances where I was using `{ ...response }`. Thanks for catching this and also good catch on the ada redemption parameters. This PR is ready for another review/merge!